scripts/check_formatting.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```

Comment:
```
#!/usr/bin/env sh
```

Would it be better, because it's cross platform compatible.

Reply:
`/bin/sh` is guaranteed to exist on every POSIX-like system and unless you’re running Solaris it’s going to be a fairly recent POSIX-shell.

Reply:
In all other scripts in that directory we use
```
#!/usr/bin/env bash
```
It may be worth using the same shebang in too .sh files